### DISCUSSION TITLE
Image prompting

### DISCUSSION THREAD

**Discussion by EliasBL** [Genesis-Embodied-AI/genesis-world#278] [Q&A]
Hi, I'd like to use Genesis for a specific usecase but I need to prompt it via text + image. Is this possible or will it be possible?

**Comment by JocelynVelarde** [comment]
do you mean using the `gs.generate()` command?

**Reply by EliasBL** [reply]
> do you mean using the `gs.generate()` command?

Yes. Ideally IMAGE + TEXT PROMPT for context.

Is that possible?

**Reply by JocelynVelarde** [reply]
> > do you mean using the `gs.generate()` command?
> 
> Yes. Ideally IMAGE + TEXT PROMPT for context.
> 
> Is that possible?

Yea so that feature is still not available for the community. So you can't actually try it out. 😓
